docs/privacy-policy.md

```diff
@@ -0,0 +1,127 @@
+to Numfocus, usage statistics are aggregated and made available to the public.
+
+When do we share your information with others?
+----------------------------------------------
+
+When we are fulfilling our mission of being open, we will share releases of non-personal 
+information to make our products better and foster an open source software development 
+community. We will only share non-personal information and try to disclose it in a way 
+that minimizes the risk of you being identified.
+
+We will also share information when the law requires it. We follow the law whenever we 
+receive requests about you from a government or related to a lawsuit. We'll notify you 
+when we're asked to hand over your non-personal information in this way, unless we're 
+legally prohibited from doing so. Nothing in  this policy is intended to limit any legal 
+defences or objections that  you may have to a third party's request to disclose your 
+information.
+
+When we believe it is necessary to prevent harm to you or someone else we may release 
+your non-personal information. We will only share your information in this way if we 
+have a good faith belief that it is reasonably necessary to protect the rights, 
+property or safety of you, our other users, Sempervirens, Numfocus or the public.
+
+How do we store and protect your information?
+-------------------------------------------------
+
+We are committed to never storing personal information. 
+
+We are committed to protecting your non-personal information once we have it. 
+We implement physical, business, and technical security measures as needed. 
+Despite our efforts, if we learn of a security breach, we'll notify you so that you 
+can take appropriate protective steps.
+
+How will we notify you?
+------------------------
+
+Since no personal information is ever stored, no direct contact with you will ever be 
+initiated. Notifications will be issued via public broadcasts such as via twitter, our 
+website, or our mailing list.
+
+What else should you know?
+---------------------------
+
+We're  a global organization and our computers are in several different places around 
+the world. We also use service providers whose computers may also be in various 
+countries. This means that your information might end  up on one of those computers 
+in another country, and that country may have a different level of data protection 
+regulation than yours. By  giving us information, you consent to this kind of transfer 
+of your information. No matter what country your information is in, we comply with 
+applicable law and will also abide by the commitments we make in  this privacy policy.
+
+If you are under 13, we don't want your information, and you must not 
+provide it to us. If you are a parent and believe that your child who is under 13 
+has provided us with personal or non-personal information, please contact us. 
```

Comment:
What is the purpose of asking parents to contact us in this case? It seems like there's nothing we could do about it. Do we need to implement a mechanism for people to retrospectively remove their data using the installation ID?